tests/foreman/installer/test_infoblox.py

```diff
@@ -0,0 +1,153 @@
+"""Tests for Infoblox Plugin
+
+@Requirement: Infoblox
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: Installer
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import stubbed, tier3
+from robottelo.test import TestCase
+
+
+class InfobloxTestCase(TestCase):
+    @stubbed()
+    @tier3
+    def test_update_appliance_credentials(self):
+        """Check infoblox appliance credentials are updated
+
+        @id: 2e84a8b4-79b6-11e6-8bf8-68f72889dc7f
+
+        @Steps:
+
+        1. Pass appliance credentions via installer
+
+        @Assert: config/dns_infoblox.yml should be updated with
+        infoblox_hostname, username & password
+
+        @caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier3
+    def test_enable_dns_plugin(self):
+        """Check Infoblox DNS plugin can be enabled on host
+
+        @id: f8be8c34-79b2-11e6-8992-68f72889dc7f
+
+        @Steps:
+
+        1. Enable Infoblox plugin via installer
+
+        @Assert: Check DNS plugin is enabled on host
+
+        @caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier3
+    def test_disable_dns_plugin(self):
+        """Check Infoblox DNS plugin can be disabled on host
+
+        @id: c5f563c6-79b3-11e6-8cb6-68f72889dc7f
+
+        @Steps:
+
+        1. Disable Infoblox plugin via installer
+
+        @Assert: Check DNS plugin is disabled on host
+
+        @caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier3
+    def test_enable_dhcp_plugin(self):
+        """Check Infoblox DHCP plugin can be enabled on host
+
+        @id: 75650c06-79b6-11e6-ad91-68f72889dc7f
+
+        @Steps:
+
+        1. Enable Infoblox plugin via installer
+
+        @Assert: Check DHCP plugin is enabled on host
+
+        @caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier3
+    def test_disable_dhcp_plugin(self):
+        """Check Infoblox DHCP plugin can be disabled on host
+
+        @id: ea347f34-79b7-11e6-bb03-68f72889dc7f
+
+        @Steps:
+
+        1. Disable Infoblox plugin via installer
+
+        @Assert: Check DHCP plugin is disabled on host
+
+        @caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier3
+    def test_set_dhcp_provider(self):
+        """Check Infoblox DHCP plugin is settled as dhcp provider
+
+        @id: 23f76fa8-79bb-11e6-a3d4-68f72889dc7f
+
+        @Steps:
+
+        1. Set infoblox as dhcp provider with param
+        --foreman-proxy-dhcp-provider='infoblox'
+
+        @Assert: Check inflobox is DHCP provider
+
+        @caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier3
+    def test_ip_range(self):
+        """Check Infoblox ip range is used when provisioning host is done while
+        Infloblox is settled as DHCP provider
```

Comment:
Description can be defined as """Check host get IP from Infoblox IP range while provisioning a host""